lte/gateway/c/core/oai/tasks/amf/amf_as.cpp

```diff
@@ -498,6 +498,27 @@ int amf_reg_acceptmsg(const guti_m5_t* guti, const tai_t* tai,
       .unit = 0;
   nas_msg->security_protected.plain.amf.msg.registrationacceptmsg.gprs_timer
       .timervalue = 6;
+  nas_msg->security_protected.plain.amf.msg.registrationacceptmsg
+      .network_feature.iei = 0x21;
```

Comment:
Check the possibility of defining Macoros in NAS5G library.

Reply:
Comment addressed.